library/Vanilla/Web/Page.php

```diff
@@ -110,21 +117,32 @@ abstract class Page implements InjectableInterface, CustomExceptionHandler {
      * @param WebpackAssetProvider $assetProvider
      * @param BreadcrumbModel $breadcrumbModel
      * @param ContentSecurityPolicyModel $cspModel
+     * @param ConfigurationInterface $config
      */
     public function setDependencies(
         SiteMeta $siteMeta,
         \Gdn_Request $request,
         \Gdn_Session $session,
         WebpackAssetProvider $assetProvider,
         BreadcrumbModel $breadcrumbModel,
-        ContentSecurityPolicyModel $cspModel
+        ContentSecurityPolicyModel $cspModel,
+        ConfigurationInterface $config
     ) {
         $this->siteMeta = $siteMeta;
         $this->request = $request;
         $this->session = $session;
         $this->assetProvider = $assetProvider;
         $this->breadcrumbModel = $breadcrumbModel;
         $this->cspModel = $cspModel;
+        $this->config = $config;
+
+        if ($favIcon = $this->config->get("Garden.FavIcon")) {
+            $this->setFavIcon(Gdn_Upload::url($favIcon));
+        }
+
+        if ($mobileAddressBarColor = $this->config->get("Garden.MobileAddressBarColor")) {
+            $this->setMobileAddressBarColor($mobileAddressBarColor);
+        }
```

Comment:
This is very generic core class. I don't like the idea to pollute with new method and properties every time when we need to add some `<meta>` tag. We have `addMetaTag()` method which is enough to achieve the goal here and to avoid necessity to create 2 new methods.

And if we go with `addMetaTag()` we don't need to touch twig. It will reflect automatically.
What do you think?

Reply:
Seems like a sane update to me!